app/controllers/styles_hera_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StylesHeraController < AuthenticatedController
```

Comment:
Does this need to be called StylesHereController?
Can we rename it to HeraController instead and have hera/index? or styles/hera/index

Reply:
This is carried over from `StylesTyliumController`. Happy to change this but it's out of scope here.